models/new.sql

```diff
@@ -0,0 +1,5 @@
+{{ config(materialized='table',transient=true)}}

```

Comment:
Please follow uniform coding standards . like here for spacing . 
`{{ config(materialized='table', transient=true) }}`